team/user_journey.md

```diff
@@ -0,0 +1,7 @@
+## User Journey
+
+- As a student, I can look up my classes so that I can quickly learn where my classrooms are
```

Comment:
Would probably be better to use user instead of student as the platform can also benefit faculty, families, and visitors.